lib/core/macros.nim

```diff
@@ -499,8 +499,10 @@ proc bindSym*(ident: string | NimNode, rule: BindSymRule = brClosed): NimNode {.
 
 proc genSym*(kind: NimSymKind = nskLet; ident = ""): NimNode {.
   magic: "NGenSym", noSideEffect.}
-  ## Generates a fresh symbol that is guaranteed to be unique. The symbol
-  ## needs to occur in a declaration context.
+  ## Generates a fresh symbol.
+  ## if `ident` is not empty, the generated symbol maybe ambiguous.
```

Comment:
No, it's really always unique, it's just that the string representation of the code might not reflect this.

Reply:
So users need to understand symbols but your documentation patch makes it worse.

Reply:
okay then, someone may have good one in mind.